src/Compilers/Helpers/ShadowCopyAnalyzerAssemblyLoader.cs

```diff
@@ -75,6 +75,9 @@ private void DeleteLeftoverDirectories()
             }
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods", 
+            MessageId = "System.Reflection.Assembly.LoadFrom",
+            Justification = "We need to call Assembly.LoadFrom in order to load analyzer assemblies.")]
```

Comment:
@shyamnamboodiripad I sent you an email with a more detailed justification. Perhaps too detailed; feel free to summarize it. :-)